ui/src/main/webapp/home/index.js

```diff
@@ -129,7 +130,7 @@ const TileLink = ({ to, title, subtitle, children }) => (
             style={{width: '100%', height: '100%'}}>
 
             <p className={styles.titleTitle}>{title}</p>
-            {children}
+            {React.createElement(Icon, { style: { color: muiTheme.palette.primary1Color, width: '50%', height: '50%' } })}
```

Comment:
We needed to apply additional styles directly to the Icon being passed in so it's color scheme matched the theme's primary1Color. This line essentially has the same effect as if we kept `{children}` here and every time we used `TileLink` we gave it an already muiTheme connected `<Icon style={{ color: muiTheme.palette.primary1Color, width: '50%', height: '50%' }} />`. We didn't want to force all calling instances to manually muiTheme connect their children, so this adds in the styles manually within the TileLink component.